rust/Dockerfile

```diff
@@ -125,16 +125,21 @@ ENV PACKAGE=${PACKAGE}
 CMD $PACKAGE
 
 # used for local development
-FROM runtime as development
+FROM runtime as testing
```

Comment:
```suggestion
FROM runtime as test
```

Reply:
If we call it `testing` next staged should be `debugging` and `releasing` for consistency, so better just call it `test`.

Reply:
Fixed